django_tidb/features.py

```diff
@@ -0,0 +1,438 @@
+#
```

Comment:
We don't need a blank line. Ditto others.

Reply:
Done